app/controllers/samples_controller.rb

```diff
@@ -17,37 +18,6 @@ def new
     @sample = Sample.new
   end
 
-  # GET /samples/insert
-  def insert
-    # TODO: merge create and insert somehow
-    params.require([:sample_name, :project_name, :s3_input_path])
-    @project = Project.find_by(name: params[:project_name]) ||
```

Comment:
how does def create figure out which project this sample belongs to?

Reply:
the payload includes a project_id field